### DISCUSSION TITLE
Road speed limits: no sign in map (big overlay)

### DISCUSSION THREAD

**Discussion by ThomasMeister123** [osmandapp/OsmAnd#13745] [Q&A] [answered]
Hello. I miss the "real" traffic sign (big overlay) as shown here: https://docs.osmand.net/assets/images/widgets/ex_warning_speed.png

While navigation the speed limit is shown in the small panel top right. Also the speed limit ist announced by voice (where you are above the limit)). But where can I "switch on" o show then "real" traffic sign?

**Comment by mariush444** [comment, marked as answer]
navi settings. -> screen alerts -> traffic warnings   should be on
